SecretSanta/src/SecretSanta.Api/Controllers/PairingsController.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using SecretSanta.Api.ViewModels;
+using SecretSanta.Domain.Models;
+using SecretSanta.Domain.Services;
+using SecretSanta.Domain.Services.Interfaces;
+
+namespace SecretSanta.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PairingsController : ControllerBase
+    {
+        private IPairingService PairingService { get; set; }
+        private IMapper Mapper { get; set; }
```

Comment:
Because these properties are only set in the constructor they can be made readonly.
```suggestion
        private IMapper Mapper { get; }
```